### PR TITLE
[WIP] Linux Install Documentation

### PR DESCRIPTION
**Pull request by lfield**
I don't have permissions for the wiki and thought doing a PR would enable us to discuss this.

### CODE REVIEW

File: doc/linux_install.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Installing on Linux
+The recomended way to install the BOINC client on Linux is to use the package management system of your Linux distribution. If the BOINC client does not exist for your distribtion, please [open an issue](https://github.com/BOINC/boinc/issues/new) in github. Note that there are two packages, one for BOINC Client and another for BOINC Manager, which can be installed separately. Only the client is required but it is likely that the manager is wanted as well unless the client is to be managed from a remote host.
+## Debian
+Open a terminal and enter the following command:
+```
+sudo apt-get install boinc-client boinc-manager
+```
+## Ubuntu
+Use the software centre to insatll the BOINC manager or following instructions for Debian.
```
Comment (AenBleidd) [inline doc/linux_install.md]: typo

**Comment by RichardHaselgrove** [comment]
User Mark James (TarotApprentice) posted this to the alpha mailing list this morning, referring to Debian packages.

apt list boinc-client* -a
Listing... Done
boinc-client/unstable 7.9.2+dfsg-1 amd64 [upgradable from:7.8.4+dfsg-1\~bpo9+1]
boinc-client/stretch-backports,now 7.8.4+dfsg-1~bpo9+1 amd64 [installed,upgradable to: 7.9.2+dfsg-1]
boinc-client/stable 7.6.33+dfsg-12 amd64

boinc-client-nvidia-cuda/unstable,now 7.9.2+dfsg-1 amd64 [installed]
boinc-client-nvidia-cuda/stretch-backports 7.8.4+dfsg-1\~bpo9+1 amd64
boinc-client-nvidia-cuda/stable 7.6.33+dfsg-12 amd64

boinc-client-opencl/unstable 7.9.2+dfsg-1 amd64
boinc-client-opencl/stretch-backports 7.8.4+dfsg-1\~bpo9+1 amd64
boinc-client-opencl/stable 7.6.33+dfsg-12 amd64

As you can see Debian claims there are 3 packages, with different versions in different repos.

**Comment by lfield** [comment]
This is normal. You would expected unstable (testing) and stable to have different versions. Why 7.8.4 is in backport rather than stable, I don't know, but at long as apt-get picks the latest version, it should be fine.

**Comment by RichardHaselgrove** [comment]
I was more pointing out the availability of the meta-packages which include the installation of (presumably BOINC-compatible) NVidia or ATI drivers in the same operation - thus making the complete installation of BOINC on such hardware simpler.

**Comment by lfield** [comment]
I didn't spot that. This is will be for the GPU applications and isn't mentioned on the original install guide. I haven't played around with it myself either.

**Comment by RichardHaselgrove** [comment]
I hadn't heard about them either, until I asked Mark a silly question last night!

**Comment by RichardHaselgrove** [comment]
There are also user-generated resources out there. I've just been pointed to

http://markjatboinc.blogspot.co.uk/2017/04/setting-up-linux-cuda-cruncher.html

**Comment by lfield** [comment]
This suggest an issue with collaboration. Why did this volunteer not help to improve the BOINC documentation?

**Comment by RichardHaselgrove** [comment]
That's a good question - we'll have to ask him. This user was a fairly prolific contributor to the boinc_alpha mailing list under the handle 'TarotApprentice' throughout the years 2009-2014 (and possibly earlier - that's as far back as my readily accessible records go), but has been quiet since then. His last posts concerned the v7.4 development cycle, which was the last Linux build made publicly available via the BOINC download page.

I have a feeling (but with no hard evidence) that there is a large community-led BOINC environment, largely disconnected from (and invisible to) 'upstream': it would include project message boards, boinc team websites (especially national teams for non-English speaking nations), specialist hardware groupings, and probably more. This community activity is more concerned with problem-solving, tuning for performance, and coalescing around, in this case, specific Linux distributions - especially the more recent consumer distros targetted at 'anything but Windows' refugees. It doesn't easily generalise to consider issues like cross-platform coding standards.

This particular blog was first posted at SETI (https://setiathome.berkeley.edu/forum_thread.php?id=81271&postid=1861619#1861619), before being worked up for wider distribution in stand-alone form.

**Comment by lfield** [comment]
I think that we have to get the message out that we are now a community and everyone can contribute. The wiki should either be more open like Wikipedia or we should move the documentation to git. I really like the online features of git for editing documentation. You can create new files, edit files and automatically create PR all through the Web.

**Comment by Ageless93** [comment]
Opening the Wiki means we'll get spam galore.


-- Jord van der Elst.

**Comment by lfield** [comment]
How does Wkipedia deal with guest edits then? Are they not both powered by MediaWiki?

**Comment by RichardHaselgrove** [comment]
I suspect they have a lot of bots monitoring changes. When I had to update an obscure private page, it was picked up by a moderator much quicker than it would have been with reactive moderation only.

**Comment by davidpanderson** [comment]
A problem with making docs publicly editable is that there are many
well-meaning people
whose contributions are some combination of verbose, folksy, unclear, and
misinformed.
Cleaning it up is way more work than writing it yourself.

**Comment by lfield** [comment]
The wiki has been updated
https://boinc.berkeley.edu/wiki/Installing_BOINC#Linux

**Pull request closed without merging**